site/_posts/2017-01-10-blog-about-all-the-things.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Last Week in ManageIQ: Blog about all the things"
+author: syncrou
+date: 2017-01-10 09:20:19 -05:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi! [syncrou][me] here to continue our discussions as to what happened in the ManageIQ world last week. As the hours passed by and minutes zipped on,
+we all thought that a week couldn't possibly justify a blog post. Au contraire, this post is about our ability to [manage all the things][manageiq]!
+
+In the last week we not only continued to manage more things, we did it better! While some thought that there was room for
+[improvement][issues], others (everyone) agreed on one overarching theme. As long as infrastructure and clouds and subnets and networks and hybrid clouds
+and computers and phones and IFTT and general things based on technology exist ([breathe][breath]),  
```

Comment:
@jrafanie - If I'm not being overly wordy - then something is clearly and totally wrong! - I'll get it changed.